iterators task (1)/student/include/impl.h

```diff
@@ -5,11 +5,46 @@ template <class Container>
 struct accumulator
 {
     /** @todo Iterator traits*/
+    using traits = std::iterator_traits<typename Container::iterator>;
+    using difference_type = typename traits::difference_type;
```

Comment:
does it has sense?
for output iterators, all properties are usually defined as void

Reply:
correct